README.md

```diff
@@ -217,12 +219,13 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
 [AWS]: https://aws.amazon.com/
 [Semantic Versioning (SemVer)]: https://semver.org/
 
-[Github teams](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/organizing-members-into-teams)
-
+<!-- markdown-link-check-disable -->
 [variables.tf]: https://github.com/mineiros-io/terraform-github-team/blob/master/variables.tf
 [examples/]: https://github.com/mineiros-io/terraform-github-team/tree/master/examples
 [Issues]: https://github.com/mineiros-io/terraform-github-team/issues
 [LICENSE]: https://github.com/mineiros-io/terraform-github-team/blob/master/LICENSE
 [Makefile]: https://github.com/mineiros-io/terraform-github-team/blob/master/Makefile
 [Pull Requests]: https://github.com/mineiros-io/terraform-github-team/pulls
 [Contribution Guidelines]: https://github.com/mineiros-io/terraform-github-team/blob/master/CONTRIBUTING.md
+[Github teams]: (https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/organizing-members-into-teams)
```

Comment:
these links are all public, so remove the mardown-link-check. 

To test locally after removing you can just run `make test/pre-commit`

Reply:
The () need to be removed

Reply:
good catch :D

Reply:
🦅